compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -174,8 +174,7 @@ FailureOr<ParameterSetting> ParameterSetting::create(linalg::LinalgOp linalgOp,
   // the element types of all tensors which need to be allocated in memory
   // simultaneously.
   FailureOr<unsigned> maybeScaleFactor =
-      isObjectFifo ? getTilingScaleFactor(initType.getElementType())
```

Comment:
We should still take the initType into account to be safe. To improve the tile sizes safely, while also making them as large as possible, we should compute it based on all three (`lhs`, `rhs`, `init`) and whether or not double buffering is enabled for L1.